src/AdminTab/TableList.tsx

```diff
@@ -41,6 +41,15 @@ const TableDropdown: FunctionComponent<TableDropdownProps> = ({ table, onOpenMet
       >
         {t('New SQL query')}
       </Menu.Item>
+      <Menu.Item
+        icon={<ConsoleSqlOutlined />}
```

Comment:
Not necessarily now, but let's remember to change the icon (we can use the chart one we had before)

Reply:
Ah yes, lazy copy and paste to just get something running.